### PR TITLE
STYLE: Reorganize wrapping files to ease PA3DSCI migration to ITK core

### PR DESCRIPTION
**Pull request by dzenanz**
This is a follow-up to #232.

### CODE REVIEW

File: wrapping/itkImageUltrasound.wrap [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 itk_wrap_include("list")
 itk_wrap_include("itkIndex.h")
+itk_wrap_include("itkSpectra1DSupportWindowToMaskImageFilter.h")
```
Comment (tbirdso) [inline wrapping/itkImageUltrasound.wrap:7]: Is this used in this file?

**Comment by dzenanz** [comment]
Moving PA3DSCI into ITK core produces:
```log
212>------ Build started: Project: itk-stub-files, Configuration: RelWithDebInfo x64 ------
212>Generating .pyi files for Python wrapper interface
212>Traceback (most recent call last):
212>  File "C:\Dev\ITK-git\Wrapping\Generators\Python\itk\pyi_generator.py", line 609, in <module>
212>    class_name = unpack(class_files, options.pyi_dir)
212>                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
212>  File "C:\Dev\ITK-git\Wrapping\Generators\Python\itk\pyi_generator.py", line 391, in unpack
212>    base = merge(class_definitions)
212>           ^^^^^^^^^^^^^^^^^^^^^^^^
212>  File "C:\Dev\ITK-git\Wrapping\Generators\Python\itk\pyi_generator.py", line 450, in merge
212>    raise AssertionError(
212>AssertionError: The basic values of two associated classes do not match.
212>Item 1: class `itkImageFileReader::ImageFileReader` has_new_method: `True` typed: `True` is_abstract: `False` is_enum: `False` has_superclass: `False`Item 2: class `itkPhasedArray3DSpecialCoordinatesImageFilters::ImageFileReader` has_new_method: `True` typed: `True` is_abstract: `False` is_enum: `False` has_superclass: `True`
212>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets(248,5): error MSB8066: Custom build for 'C:\Dev\ITK-py11\CMakeFiles\5111c4c6071a90c7f914b45552b82377\itk-stub-files.rule' exited with code 1.
212>Done building project "itk-stub-files.vcxproj" -- FAILED.
```
which made me change `itk_wrap_class("itk::ImageFileReader" POINTER_WITH_SUPERCLASS)` into `itk_wrap_class("itk::ImageFileReader" POINTER)`. Should we make the similar changes for other filters in Ultrasound, such as `CurvilinearArraySpecialCoordinatesImage`, `CastImageFilter`, `RescaleIntensityImageFilter` etc?

**Comment by dzenanz** [comment]
I just pushed my [current state](https://github.com/dzenanz/ITK/commit/d7127a3b87592258e7a8644b3d0e8b2f47d9cc6c) to my fork's master. This includes the change to avoid the above error.

**Comment by dzenanz** [comment]
I think the reason for not seeing this error earlier, is that now both of those wrappings are part of the same translation unit (or better to say, wrapping unit).

**Comment by dzenanz** [comment]
So maybe these differences don't matter, unless both definitions are part of the same "wrapping unit". It would be quite a bit of effort to harmonize them all, and if we miss some we would not get any warnings. So my question is: does it matter? @thewtex could pitch in too.

**Comment by tbirdso** [comment]
> I think the reason for not seeing this error earlier, is that now both of those wrappings are part of the same translation unit (or better to say, wrapping unit).

Ah, good point. Yes, in general I agree that the wrapping specification (`POINTER`, `POINTER_WITH_SUPERCLASS`, etc) in ITKUltrasound should match the base class in ITK.

**Comment by dzenanz** [comment]
I just created a draft PR https://github.com/InsightSoftwareConsortium/ITK/pull/4099. We should split the discussion between this PR and that one, as appropriate.

**Comment by dzenanz** [comment]
Closed in favor of #234.

**Pull request closed without merging**